content/en/docs/setup/install/multiple-controlplanes/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Install Multiple Istio Control Planes in a Single Cluster
+description: Install multiple Istio control planes in a single cluster using revisions and discoverySelectors.
+weight: 55
+keywords: [multiple,control,istiod,local]
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+This guide walks you through the process of installing multiple Istio control planes within a single cluster and
+then a way to scope workloads to specific control planes. For this guide, [the deployment model](/docs/ops/deployment/deployment-models/#control-plane-models)
+will have a single Kubernetes control plane with multiple Istio control planes and multiple meshes.
+The separation between the meshes is provided by Kubernetes namespaces and RBAC.
+
+{{< image width="80%"
+    link="single-cluster-multiple-istiods.svg"
```

Comment:
Nit: the horizontal blue dashed lines look pretty sloppy in this diagram, because they don't line up.

Reply:
Updated the document, thanks for the help :)

Reply:
One more observation is that there is a lot of blank space above and below the diagram in the doc preview. You can eliminate it by cropping away the excess canvas in the google diagram before generating the svg.

Reply:
Have tried to crop the image